lib/libutee/tee_api_objects.c

```diff
@@ -237,21 +293,59 @@ void TEE_InitValueAttribute(TEE_Attribute *attr, uint32_t attributeID,
 	attr->content.value.b = b;
 }
 
+/*
+ * Use of this function is deprecated
+ * new code SHOULD use the TEE_CopyObjectAttributes1 function instead
+ * These functions will be removed at some future major revision of
+ * this specification
+ */
 void TEE_CopyObjectAttributes(TEE_ObjectHandle destObject,
 			      TEE_ObjectHandle srcObject)
 {
 	TEE_Result res;
+
+	res = TEE_CopyObjectAttributes1(destObject, srcObject);
```

Comment:
This change is not 1.1 compliant:
*If the source object is corrupt then this function copies no attributes and leaves the target object uninitialized.*